test/bdd/steps/lib/utilities.js

```diff
@@ -5,6 +5,34 @@ function calculateImportHash(data) {
     return `0x${sha3_256(sortedStringify(data, null, 0))}`;
 }
 
+/**
+ * Normalizes hex number
+ * @param number     Hex number
+ * @returns {string} Normalized hex number
+ */
+function normalizeHex(number) {
```

Comment:
can we use `normalizeHex` from `Utilities`?

Reply:
Nope, it's in the different scope.